runtime/filetype.vim.rej

```diff
@@ -0,0 +1,15 @@
+--- runtime/filetype.vim
```

Comment:
`*.rej` files should not be committed.

```
-" Dockerfilb; Podman uses the same syntax with name Containerfile
+" Dockerfile; Podman uses the same syntax with name Containerfile
```
suggests that an earlier runtime patch is required but it's fine. These typo changes happen a lot.

Reply:
Oh ok gonna fix it wait a sec ^